modules/i18n/input-method/fcitx5.nix

```diff
@@ -27,6 +29,8 @@ in {
       GLFW_IM_MODULE = "ibus"; # IME support in kitty
       GTK_IM_MODULE = "fcitx";
       QT_IM_MODULE = "fcitx";
+      QT_PLUGIN_PATH =
+        "$QT_PLUGIN_PATH:${fcitx5Package}/${qtbase.qtPluginPrefix}:${fcitx5Package}/${qt6.qtbase.qtPluginPrefix}";
```

Comment:
Do we need both versions? The NixOS module has only qt6. (I know nothing about Qt)